woocommerce/woocommerce
# Update DOMPurify to 2.5.7

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This updates DOMPurify to 2.5.7 to receive the latest security patches.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the new product editor at `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`
2. Disable tax rates and calculations at `/wp-admin/admin.php?page=wc-settings&tab=general`
3. Go to Products > Add New
4. Verify that "Regular price" help is rendered correctly (screenshot below)

<img width="446" alt="image" src="https://github.com/user-attachments/assets/9a972076-11e7-4d88-bf4e-75531eb69df0">




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type



-   [] Update - Update existing functionality

#### Message 
Update dompurify to 2.5.7

</details>

<details>

<summary></summary>

#### Comment 

</details>
